{{cookiecutter.repo_name}}/Makefile

```diff
@@ -0,0 +1,46 @@
+.DEFAULT_GOAL := validate
```

Comment:
At one point, we had an agreement about minimal Make targets.  I thought "test" was one of them, and that it was the default target.  Shouldn't we have test here at least?